src/react-queries/queries/mutations.ts

```diff
@@ -28,7 +28,7 @@ export const mutations = createMutationKeys('mutations', {
   }) => ({
     // note how we don't add the wallet here because of its non-serializable nature
     mutationKey: ['approve', { assetId, spender, amount, from }],
-    mutationFn: async () => {
+    mutationFn: async (_: void) => {
```

Comment:
Why?

Reply:
To anyone expending this comment and wondering why: https://github.com/shapeshift/web/pull/7020#discussion_r1629382842

Removing the explicit undefined in mutation call requires the `mutationFn` to have an explicit void param